Form/Type/CollectionType.php

```diff
@@ -24,14 +23,15 @@ class CollectionType extends AbstractType
 {
     /**
      * {@inheritdoc}
+     *
+     * @todo BC code for Symfony < 2.8 . To be removed.
      */
     public function getParent()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
-            return 'Symfony\Component\Form\Extension\Core\Type\CollectionType';
-        } else {
-            return 'collection';
-        }
+        return
+            method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions') ?
```

Comment:
Thanks @stof ! This is why we need an abstraction around this. Now I have to search and replace all this…